README.md

```diff
@@ -144,6 +145,7 @@ There is also a generator which can help get you started `rails generate rspec:s
           tags 'Blogs', 'Another Tag'
           produces 'application/json', 'application/xml'
           parameter name: :id, in: :path, type: :string
+          request_example value: { some_field: 'Foo' }, name: 'basic', summary: 'Request example description'
```

Comment:
The method is `request_body_example` and not `request_example` so either README has wrong name or the method. Not sure what was the intention. `request_body_example` is probably more intuitive.

Reply:
Thanks @bblimke, you're right!

https://github.com/rswag/rswag/pull/566